source/includes_main/_organizations.md

```diff
@@ -103,6 +103,9 @@ created_at | *NULL* | ISO 8601 format with timezone offset
 updated_at | *NULL* | ISO 8601 format with timezone offset
 
 
+**Note:** This method accepts an optional URL parameter `return_existing`, which can be set to true (default=false), 
+that will return the existing organization (with HTTP status code = 200) if an organization with that name already exists. 
+
```

Comment:
I would go for the following:

"This method accepts an optional URL parameter `return_existing`. When set to `true` (default is `false`), 
that will return an existing organization if an organization with that name already exists (with `200` HTTP status code, 4XX otherwise)."

Check what's the value of the status code if it does not pass the validation. If this is `422` as expected, it has to be added to the errors section (currently missing).

Reply:
Take the occasion to correct the following double typo in the errors section:
`Service Unavailable – We’re temporarially offline for maintanance. Please try again later.` => `Service Unavailable – We’re temporarily offline for maintenance. Please try again later.`

Reply:
And add this block in the prospects section as well ;)